scripts/lighthouseRunner.js

```diff
@@ -0,0 +1,117 @@
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+
+const thresholdTypes = [
+  'acessibility',
+  'seo',
+  'best-practices',
+  'pwa',
+  'performance',
+];
+
+if (process.argv.length < 3) {
+  process.on('exit', () => {
+    console.log(`Supply thresholds for: ${thresholdTypes.join(', ')}`);
+  });
+  process.exit(9);
```

Comment:
`9 - Invalid Argument - Either an unknown option was specified, or an option requiring a value was provided without a value.` 

https://nodejs.org/api/process.html#process_event_exit

Moot point because we can pass in configs as a file.